mrtt-ui/src/App.js

```diff
@@ -2,21 +2,25 @@ import React from 'react'
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom'
 import Home from './views/Home'
 import ProjectDetailsForm from './components/ProjectDetailsForm/ProjectDetailsForm'
+import { ThemeProvider } from '@mui/material/styles'
+import theme from './styles/theme'
 
 function App() {
   return (
-    <Router>
-      <div className='app'>
-        <Routes>
-          <Route
-            path='/'
-            element={[
-              <Home key='1'></Home>,
-              <ProjectDetailsForm key='2'></ProjectDetailsForm>
-            ]}></Route>
-        </Routes>
-      </div>
-    </Router>
+    <ThemeProvider theme={theme}>
+      <Router>
+        <div className='app'>
```

Comment:
Also maybe outta scope, but for conversation: Should we style stuff with css-in-js and [Emotion](https://emotion.sh/docs/styled) Which is the default styling engine for MUI and very similar to the standard Styled Components that we have been using for many projects?

Reply:
and avoid classnames?

Reply:
hmm I think styled components is more flexible from Emotion (just based on what I've read). In the past i've used makeStyles/useStyles with mui which I see is now deprecated. I'm wondering how we will use basic layout CSS with MUI now ... looking into it!

Reply:
ok figured it out, need to do some restructuring. The new way to is it is with `styled()` . [More info here](https://mui.com/system/styled/) and [here.](https://stackoverflow.com/questions/69860893/how-to-properly-use-styled-with-mui-v5)

Reply:
Here's an example of using `styled()`: https://github.com/globalmangrovewatch/gmw-users/blob/309268cd4954dd93ecfd31903b87107b52ddfab0/mrtt-ui/src/components/ProjectDetailsForm/ProjectDetailsForm.js#L26

Reply:
Looked at this again and I was wrong.
`import { styled } from '@mui/material/styles'` like you're already doing is the way to go 🙂